gammapy/maps/base.py

```diff
@@ -265,6 +265,24 @@ def sum_over_axes(self):
         """Reduce to a 2D image by summing over non-spatial dimensions."""
         pass
 
+    def coadd(self, map_in):
+        """Fill this map with the contents of another map.  This method can be
```

Comment:
I find "fill" here slightly confusing. Can you change the first line to more clearly say that this "adds" and doesn't "overwrite" the content?

Reply:
Elsewhere "fill" is used to mean increment the value (e.g.  the `fill_by` methods). I can change to "Add the contents of `map_in` to this map."

Reply:
What is the use case of the coadd method for maps with different projections? Shouldn't it be possible only if geometries are aligned and bin size equal? Summing counts this way makes sense only if the final bin size is larger than that of ``map_in``.

Reply:
The most common use-case would be for maps that have the same projection.  In that case one can combine counts maps as long as the bin size of `map_in` is an integer multiple of the map bin size.  If you're worried that people may misuse this method we could consider putting in some check to ensure the geometries are compatible.